### DISCUSSION TITLE
Change the volume in embedded videos.

### DISCUSSION THREAD

**Discussion by Olizja** [Araxeus/Youtube-Volume-Scroll#2] [Q&A] [answered]
Hi, is it possible to change the code so that it would be possible to change the volume of embedded videos on other sites /forums?

**Comment by Araxeus** [comment]
Theoretically yes but it would require additional permissions and much code change

right now the code is only authorized to run on youtube.com and music.youtube.com domains,
if we want it to work on any embedded video it would need to run on every website

the code would also need to be changed since we would need to constantly watch for video's in a page, and also there could also be a few videos in one page

Also as of right now, I don't know if I can actually access embedded videos since they are somehow displayed in an empty iframe like so:
```html
<iframe width="356" height="200" src="https://www.youtube.com/embed/Aj43zrrQd0o?feature=oembed&amp;enablejsapi=1" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
```
> (Example from reddit youtube embed)

and I can't figure out how to even access any of the video properties or the youtube api from there

**Reply by Olizja** [reply]
Hello. Thanks for the answer
If there is any news, write.

**Comment by Araxeus** [comment, marked as answer]
@Olizja abit late there but I finally figured it out,

This feature was included in v1.3.0 
(following pr #4)
